docs/_partner_api/errors.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Errors
+order: 4
```

Comment:
`Authentication` and this both have `order: 4`. It looks like it works fine when run (per your gif) but you might replace one so that it's exactly how you want.